CHANGELOG.md

```diff
@@ -4,9 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 0.1.3 - 20178-02-08
+## 0.1.3 - 2018-02-08
 Initial public release.
 
 ### Added
 * The `ngx-uploadcare-widget` component.
-* README with info about install, usage, etc.
\ No newline at end of file
+* README with info about install, usage, etc.
+
+## 0.1.4 - 2018-03-12
```

Comment:
If you add new method, the version should change minor part, `0.2.0`.

We need to follow semver, read more about it http://semver.org/spec/v2.0.0.html